python/lsst/obs/pfs/isrTask.py

```diff
@@ -68,6 +73,66 @@ class PfsIsrTaskConfig(ipIsr.IsrTaskConfig):
 
     def validate(self):
         super().validate()
+        if not self.doSaturationInterpolation and "SAT" in self.maskListToInterpolate:  # fixed on LSST master
+            self.maskListToInterpolate.remove("SAT")
+#
+# Code to handle the broken shutter matrix matrix inverse cache
+#
+
+
+class BrokenShutterKernelCache:
+    __cache = {}
+    __maxsize = 0
+
+    def __init__(self, maxsize=0):
+        self.setCacheSize(maxsize)
+
+    @classmethod
+    def compute(cls, kern, t_exp):
```

Comment:
Missing docString.

Reply:
I think it's barely worth adding, but I'll add something that does a little more than echoing the method names.